### PR TITLE
[ESP32] Configure the event logging buffers to reduce dram overflow i…

### PR DESCRIPTION
**Pull request by shubhamdp**
…n all-clusters-app for m5stack

DRAM is overflowing... this is the temporary fix which reduces the event logging buffers and being tracked in #34717

### CODE REVIEW

**Review by andy31415 — APPROVED** [review]

**Comment by andy31415** [comment]
Hotfixing ToT build breakage.

File: examples/all-clusters-app/esp32/sdkconfig_m5stack.defaults [hunk 1]
```diff
@@ -73,3 +73,9 @@ CONFIG_BUILD_CHIP_TESTS=y
 
 # Move functions from IRAM to flash
 CONFIG_FREERTOS_PLACE_FUNCTIONS_INTO_FLASH=y
+
+# Reduce the event loggin buffer to reduce the DRAM usage
```
Comment (bzbarsky-apple) [inline examples/all-clusters-app/esp32/sdkconfig_m5stack.defaults:77]:
```suggestion
# Reduce the event logging buffer to reduce the DRAM usage
```

File: examples/all-clusters-app/esp32/sdkconfig_m5stack_rpc.defaults [hunk 2]
```diff
@@ -78,13 +78,16 @@ CONFIG_BUILD_CHIP_TESTS=y
...
-
 CONFIG_DIAG_USE_EXTERNAL_LOG_WRAP=y
 
 # Memory Optimizations
 CONFIG_NIMBLE_MAX_CONNECTIONS=1
 CONFIG_BTDM_CTRL_BLE_MAX_CONN=1
 CONFIG_BT_NIMBLE_ROLE_CENTRAL=n
 CONFIG_BT_NIMBLE_ROLE_OBSERVER=n
+
+# Reduce the event loggin buffer to reduce the DRAM overflow
```
Comment (bzbarsky-apple) [inline examples/all-clusters-app/esp32/sdkconfig_m5stack_rpc.defaults:89]:
```suggestion
# Reduce the event logging buffer to reduce the DRAM overflow
```

**Review by bzbarsky-apple — APPROVED** [review]

File: examples/all-clusters-app/esp32/sdkconfig_m5stack_rpc.defaults (hunk 2 above)
Comment (shubhamdp) [reply examples/all-clusters-app/esp32/sdkconfig_m5stack_rpc.defaults:89]: Addressed in #34729

File: examples/all-clusters-app/esp32/sdkconfig_m5stack.defaults (hunk 1 above)
Comment (shubhamdp) [reply examples/all-clusters-app/esp32/sdkconfig_m5stack.defaults:77]: Addressed in #34729

**Pull request merged**